apps/prairielearn/src/pages/instructorGradebook/instructorGradebook.sql

```diff
@@ -98,10 +100,15 @@ WITH
       u.uid,
       u.uin,
       u.name AS user_name,
+      e.id AS enrollment_id,
       users_get_displayed_role (u.user_id, $course_instance_id) AS role
     FROM
       user_ids
       JOIN users AS u ON (u.user_id = user_ids.user_id)
+      LEFT JOIN enrollments AS e ON (
+        e.user_id = u.user_id
+        AND e.course_instance_id = $course_instance_id
+      )
```

Comment:
The join on enrollments here seems unnecessary. As far as I can see we never use `course_users.enrollment_id`? We repeat the join on enrollments below (lines 158-161) and actually get the enrollment data from that join on line 154.